server/public/styles.css

```diff
@@ -0,0 +1,71 @@
+body {
+  background-color: #333;
+}
+
+.dark-color {
+  background-color: #292929;
+}
+
+.logo {
+  font-size: 2rem;
+  color: white;
+  text-shadow:
+    -1px -1px 0 red,
+    1px -1px 0 red,
+    -1px 1px 0 red,
+    1px 1px 0 red;
+  font-family:
+    Ruda,
+    sans-serif;
+}
+
+.profile-pic {
+  height: 50vh;
+  object-fit: cover;
+}
+
+.list-header {
+  color: white;
+  font-family:
+    Ruda,
+    sans-serif;
+}
+
+ul {
+  list-style: none;
+}
+
+.position,
+.location {
+  color: #8d8a8a;
+  font-size: 16px;
+  font-family:
+    Ruda,
+    sans-serif;
+}
+
+.username {
+  color: #343a40;
+  display: inline;
+}
+
+.card-img {
+  height: 30vh;
```

Comment:
I don't recommend using a relative unit for heights, try using a px value instead.